src/RSSFeed.php

```diff
@@ -327,7 +225,7 @@ public static function getVisibilityCriteria(bool $forceall = false): array
             ];
         }
 
-        if (isset($_SESSION["glpiactiveprofile"]) && isset($_SESSION["glpiactiveprofile"]['id'])) {
+        if (isset($_SESSION["glpiactiveprofile"]['id'])) {
```

Comment:
I'm not 100% sur for this one. Won't that cause an undefined index issue if `$_SESSION["glpiactiveprofile"]` is not set?

Reply:
No. Those errors are not raised when checking with `isset`.

```
php > echo isset($_SESSION['fake']['fake2']) ? 'true' : 'false';
false
```